packages/editor/src/dataviews/fields/index.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This PR is an opportunity to create a dedicated folder/file for the post type fields. If the fields declaration grows and becomes too big we can separate each field in its own file.